RocketChat/Rocket.Chat.ReactNative
# [FIX] "Sent an attachment" text on media preview for threads

## Proposed changes

When attachments show up the first time they append a text saying "Sent an attachment" on it when they don't have a description. 
## Issue(s)	


## How to test or reproduce

1. With user A, send an image to a thread
2. With user B, open the thread

You shouldn't see the `Sent an attachment` text.
## Screenshots

## Types of changes




## Checklist



## Further comments
